client/src/components/NextBookings/NextBooking.tsx

```diff
@@ -1,29 +1,27 @@
-import { Avatar, Card, CardContent, Grid, IconButton, Typography } from '@mui/material';
-import SettingsIcon from '@mui/icons-material/Settings';
+import { Avatar, Card, CardContent, Grid, Typography } from '@mui/material';
 import format from 'date-fns/format';
 import { Booking } from '../../interface/Booking';
+import HandleRequestButton from '../HandleRequestButton/HandleRequestButton';
 
 interface PropTypes {
   booking: Booking;
 }
 
 const NextBooking = ({ booking }: PropTypes) => {
   return (
-    <Card sx={{ padding: '10px 0px 30px 20px' }}>
+    <Card sx={{ padding: '10px 0px 30px 20px', minWidth: '280px' }}>
       <CardContent>
         <Grid container direction="row" justifyContent="space-between" alignItems="flex-end">
           <Typography sx={{ fontVariant: 'small-caps', fontWeight: 'bold' }}>your next booking:</Typography>
-          <IconButton>
-            <SettingsIcon />
-          </IconButton>
+          <HandleRequestButton bookingId={booking._id} />
         </Grid>
         <Typography sx={{ margin: '10px 0px' }}>
           {format(booking.start, 'd LLLL y, haaa')} - {format(booking.end, 'haaa')}
         </Typography>
         <Grid container direction="row" alignItems="center">
-          <Avatar src={`https://robohash.org/${booking.sitter.email}`}>{booking.sitter.name[0]}</Avatar>
+          <Avatar src={`https://robohash.org/${booking.otherUser.email}`}>{booking.otherUser.name[0]}</Avatar>
           <Typography sx={{ fontSize: '0.9rem', fontWeight: 'bold', marginLeft: '5px' }}>
-            {booking.sitter.name}
+            {booking.otherUser.name}
```

Comment:
Once again, I think it doesn't make too much sense to use the exact same page for both sitters and owners considering how different their stories/needs are. Would have been nice to see a page of its own designed for owners but that's my opinion 👍🏼